### PR TITLE
accept my new pull request

### PR DESCRIPTION
**Pull request by San97**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [ ] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by San97** [comment]
I still didn't get an official email stating that I'm enrolled in the
graduation ceremony and in the 2020 Batch. Will I get enrolled? Or did I
miss the chance?

**Comment by mitulsavani** [comment]
@Mohammed-Shoaib could you answer his queries too

**Comment by Mohammed-Shoaib** [comment]
@mitulsavani Thank you for allowing me to answer his queries :smile: 
@San97 You will not get an official email stating that you have been enrolled in the graduation ceremony. Rest assured, you will be enrolled for the graduation ceremony :confetti_ball:

Unfortunately, based on your PR you won't qualify for the Tier 2 swag but you will qualify for the Tier 1 swag. But don't feel sad, I am sure Tier 1 swag has some exciting goodies :fire:

In terms of merging your PR, based on #4154, the maintainers are on a break for the weekend. They need some rest too! :sleeping: Don't worry, your pull request will be merged soon :tada: 

Till then sit back, relax, and enjoy the weekend :relaxed:

File: _data/San97/San97.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Sneh Desai# No longer than 28 characters
+institution: Sardar Vallabhbhai Patel Institute of Technology, Vasad🚩 # no longer than 58 characters
```
Comment (Mohammed-Shoaib) [inline _data/San97/San97.md:3]: Remove the 🚩(flag) as it is a non-ascii character.

**Pull request closed without merging**